INN/Largo
# #1874 theme images 'upload' buttons not responding

Changes
This pull request makes the following changes:
file 
lib/options-framework/js/of-medialibrary-uploader.js 
lines 21 & 72 are using deprecated jquery functions preventing users form using the theme image loader.

Why
I assume people want to use this feature

For #1874 

Testing/Questions
[ X ] Is this PR targeting the correct branch in this repository?
[ ]

Steps to test this PR:
you should not be able to upload using the theme options currently. If you use the jquery migration plugin it will fix this issue but we don't want plugins

Additional information
INN Member/Labs Client requesting: (if applicable)
[x] Contributor has read INN’s GitHub code of conduct
[] Contributor would like to be mentioned in the release notes as: blank
[x] Contributor agrees to the license terms of this repository.
